pkg/app/managedisc/factory.go

```diff
@@ -14,30 +12,26 @@ import (
 
 // Factory creates appdisc.Updater instances based on the app name.
 type Factory struct {
-	Log            logrus.FieldLogger
-	PK             cipher.PubKey
-	SK             cipher.SecKey
-	UpdateInterval time.Duration
-	ServiceDisc    string // Address of service-discovery
+	Log         logrus.FieldLogger
+	PK          cipher.PubKey
+	SK          cipher.SecKey
+	ServiceDisc string // Address of service-discovery
 }
 
 func (f *Factory) setDefaults() {
 	if f.Log == nil {
 		f.Log = logging.MustGetLogger("appdisc")
 	}
-	if f.UpdateInterval == 0 {
-		f.UpdateInterval = skyenv.ServiceDiscUpdateInterval
-	}
 	if f.ServiceDisc == "" {
 		f.ServiceDisc = skyenv.DefaultServiceDiscAddr
 	}
 }
 
-// VisorUpdater obtains a visor updater.
-func (f *Factory) VisorUpdater(port uint16) Updater {
-	// Always return empty updater if keys are not set.
+// VisorManager obtains a visor manager.
```

Comment:
Visor manager is a bad name as it implies the visor is managed through this structure while it is just a client for updating entries for services it runs.

Reply:
Done.